services/ui-src/src/components/report/ReportPageWrapper.tsx

```diff
@@ -103,15 +104,30 @@ export const ReportPageWrapper = () => {
                     Previous
                   </Button>
                 )}
-                {parentPage.index < parentPage.childPageIds.length - 1 && (
+                {parentPage.index < parentPage.childPageIds.length - 1 &&
```

Comment:
Why are there 2 different ways of rendering the continue button?  Is it just to change the justifyContent? If so that can be handled with logic rather than duplicating the component

Reply:
Good catch! There's no need for the extra logic. It's handled with the first one now.